closure-stylesheets.iml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
No advantage.  It's an intellij project file.  
Will remove this file, and perhaps add a .gitignore for it.